generator/internal/rust/templates/crate/src/builder.rs.mustache

```diff
@@ -164,6 +170,19 @@ pub mod {{Codec.ModuleName}} {
             {{/Optional}}
             let execute = move |token: String| {
                 let mut builder = self.clone();
+                {{#HasAutoPopulatedFields}}
+                {{!
+                    If we have a new page token, this is a new request. New
+                    requests need new request IDs.
+                }}
+                {{^Optional}}
+                let initial = builder.0.request.{{Codec.FieldName}} == token;
+                {{/Optional}}
+                {{#Optional}}
+                let initial = builder.0.request.{{Codec.FieldName}}.as_deref().is_none_or(|s| s == token);
+                {{/Optional}}
+                builder.0.request = Self::auto_populate(builder.0.request, !initial);
```

Comment:
Instead of passing in `force`, we could clear the request ID field in the pagination case. Then we let `send()` do the work. That might be cleaner.